### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by marcialwushu** [marcialwushu/ReactProjetWebsite#3]
# Standards and Guidelines

## WCAG

The Web Content Accessibility Guidelines provides guidelines for creating accessible web sites.

The following WCAG checklists provide an overview:

    WCAG checklist from Wuhcag
    WCAG checklist from WebAIM
    Checklist from The A11Y Project

## WAI-ARIA 

The Web Accessibility Initiative - Accessible Rich Internet Applications document contains techniques for building fully accessible JavaScript widgets.

Note that all aria-* HTML attributes are fully supported in JSX. Whereas most DOM properties and attributes in React are camelCased, these attributes should be hyphen-cased (also known as kebab-case, lisp-case, etc) as they are in plain HTML:

--- 

https://reactjs.org/docs/accessibility.html

**Comment by marcialwushu** [comment]
https://www.boia.org/blog/what-is-a11y


https://a11yproject.com/
